docs/install/macos.rst

```diff
@@ -18,11 +18,15 @@ Install
    .. figure:: /img/copy-app.png
      :alt: 
 
-4. Start the Kolibri app from the **Launchpad**, the **Applications** folder, or **Spotlight**.
-#. Proceed with the :ref:`setup_initial` of your facility. 
+4. a) **Do NOT start the Kolibri app from the Launchpad, as it is not yet signed**. Use the **Finder** on your Mac instead to locate Kolibri app in the **Applications** folder.
+
+   b) Control-click the app icon, then choose **Open** from the shortcut menu.
+      This step will be saved as an exception to your security settings, so you are able to open Kolibri in the future by double-clicking it just as you would do with other apps.
```

Comment:
These steps are duplicated from `docs/access/macos.rst`. I would suggest replacing at least one of them with a reference.

Reply:
leaving for now. feel free to open an issue for this